systems/WikibaseRepo/05-Building_Block_View.md

```diff
@@ -4,9 +4,38 @@
 
 TBA
 
-## External Interfaces
-
-### Action API Edit Entity
+## Action API modules
+
+| Building Block                          | Responsibility                                                                              |
+| --------------------------------------- | ------------------------------------------------------------------------------------------- |
+| wbavailablebadges                       | Query available badge items                                                                 |
+| wbcheckconstraintparameters             | Check the constraint parameters of constraint statements                                    |
+| wbcheckconstraints                      | Perform constraint checks on any entity you want and returns the result                     |
+| wbcreateclaim                           | Create Wikibase claims                                                                      |
+| wbcreateredirect                        | Create Entity redirects                                                                     |
+| [wbeditentity](#action-api-edit-entity) | Create a single new Wikibase entity and modifies it with serialised information             |
+| wbformatentities                        | Format entity IDs to HTML                                                                   |
+| wbformatvalue                           | Format DataValues                                                                           |
+| wbgetclaims                             | Get Wikibase claims                                                                         |
+| wbgetentities                           | Get the data for multiple Wikibase entities                                                 |
+| wblinktitles                            | Associate two pages on two different wikis with a Wikibase item                             |
+| wblmergelexemes                         | Merge two Lexemes                                                                           |
```

Comment:
I wonder if lexeme related things should be in the same action api block?
I guess this "repo" top level block is made up of some sub blocks already perhaps?
I wonder what others think about this, as we didn't talk much around this topic yet.

Reply:
Yes, you are right, this got included by accident. On that note I also removed `wbcheckconstraintparameters` and `wbcheckconstraints` as they belong to the `WikibaseQualityConstraints` extension.